### DISCUSSION TITLE
Is there a way to programmatically trigger a prefetch?

### DISCUSSION THREAD

**Discussion by gitChristian** [remix-run/remix#5366] [General]
I have a grid of cards like in the photo with `prefetch="intent"` on each card.
Problem is, if a user clicks the heart icon to like the card, and then clicks on the card to view the prefetched link (a dashboard), then the 'like' is not registered because the link was prefetched on hover and the 'like' action is not reflected. Basically the prefetch is stale. 
Any patterns to get around this? 🙏
<img width="1285" alt="image" src="https://user-images.githubusercontent.com/9953695/216801554-74f26e6f-76c7-4f41-aa79-d51957e33220.png">

**Comment by mcansh** [comment]
https://remix.run/docs/en/1.13.0/components/prefetch-page-links
